### DISCUSSION TITLE
material-ui is too ugly can you change the skin?

### DISCUSSION THREAD

**Discussion by vaynevayne** [KevinVandy/material-react-table#651] [General]
It's too tiring to change it yourself.

**Comment by KevinVandy** [comment]
What is this supposed to mean lol? If you don't like MUI, don't use Material React Table.

You can check out Mantine React Table though

**Reply by vaynevayne** [reply]
I mean, you can customize the theme for mui instead of using the default

**Reply by KevinVandy** [reply]
read the customization guide https://www.material-react-table.com/docs/guides/customize-components
